### DISCUSSION TITLE
How to join Beta Testing?

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#585] [Q&A]
I just heard that the beta version of ToonSquid 2.0 will be coming soon, and I was wondering how one would go about getting to use the beta test version? Also, if anyone does get the beta version, could you share what is in the update?

**Comment by DinoAwakening** [comment]
Maybe through “TestFlight”, since it’s the most popular app for Beta-Testing on iOS. Although, it could be a closed beta, which would mean only a few and Keiwan has access to it xD

**Reply by inTIMidation297** [reply]
It's a closed beta. Only certain people can test it
